webpack.config.js

```diff
@@ -30,7 +30,12 @@ let config = {
                 test: /\.(scss)$/,
                 include: path.resolve(__dirname, 'src'),
                 loaders: ['style-loader', 'css-loader', 'sass-loader']
-            }
+            },
+            {
+                test: /\.mjs$/,
+                include: /node_modules/,
+                type: 'javascript/auto',
```

Comment:
What's this type about, got a link to an article or something so I can understand it?

Reply:
I just copied from this

Reply:
and its actually in the readme